coalib/results/Diff.py

```diff
@@ -438,9 +439,16 @@ def change_line(self, line_nr, original_line, replacement):
                                                replacement)
             replacement = ''.join((orig_diff + new_diff).modified)
 
-        linediff.change = (original_line, replacement)
+        linediff.change = (self._file[line_nr-1], replacement)
         self._changes[line_nr] = linediff
 
+    def change_line(self, line_nr, original_line, replacement):
+        # Use of original_line arg is deprecated
+        if original_line:
+            logging.warning('Use of orginal_line argument is deprecated. '
+                            'It is not required to provide that argument')
```

Comment:
not only the argument, also the whole function is deprecated so you don't need the if and the message needs changing